SpinCore_pp/run_field_dep_justMw.py

```diff
@@ -1,6 +1,3 @@
-from pylab import *
-from pyspecdata import *
-import os
```

Comment:
why isn't this file inside examples?

Reply:
in fact, there is one in examples with the same name!!  I'm going to stop here, and let you resolve that issue.

Reply:
okay I will remove from spincore_pp and make sure the changes are carried over to the examples version!